### DISCUSSION TITLE
乌班图24，安装之后，只能在linux浏览器访问面板，无法在同局域网下的其他ip设备下，访问面板。

### DISCUSSION THREAD

**Discussion by SyberRabbit** [v2rayA/v2rayA#1451] [Q&A] [answered]
如题

**Comment by eronez** [comment]
1. check v2raya server address(or sudo lsof -i:2017)
2. check firewall
3. The network prohibits devices on the same local area network (LAN) from connecting to each other

**Reply by SyberRabbit** [reply]
> 1. check v2raya server address(or sudo lsof -i:2017)
> 2. check firewall
> 3. The network prohibits devices on the same local area network (LAN) from connecting to each other

Firewall is turned off
Can be accessed by other web applications

**Comment by SyberRabbit** [comment, marked as answer]
我已经用反代解决了， 反正早晚也得用反代的，无法直接访问也无妨
